test/unit/math/relative_tolerance.hpp

```diff
@@ -0,0 +1,78 @@
+#ifndef TEST_UNIT_MATH_RELATIVE_TOLERANCE_HPP
+#define TEST_UNIT_MATH_RELATIVE_TOLERANCE_HPP
+
+#include <stan/math.hpp>
+#include <algorithm>
+
+namespace stan {
+namespace test {
+
+/**
+ * Struct holding information about relative tolerance and the minimal absolute
+ * tolerance that should be tested against. The final tolerance is computed as
+ * max(tol * fabs(x), tol_min)
+ * Where x is either an exact value to be tested against or average of
+ * two inexact values to be compared.
+ */
+struct relative_tolerance {
+  /**
+   * The relative tolerance
+   */
+  double tol;
```

Comment:
The convention we've adopted is that member variables get underscore suffixes and nothing else does.  So these should be `tol_` and `tol_min_` and the args to the constructor should be the ones without underscores.